config/config.exs

```diff
@@ -1,7 +1,13 @@
 use Mix.Config
 
-config :system_registry, SystemRegistry.Processor.Config,
-  priorities: [
-    :debug,
-    :nerves_network
```

Comment:
system_registry 0.4.0 sets the default priority levels so the top level config doesn't have to. It was kind of awkward that the main application would have to declare the priorities in situations where they didn't even know about system_registry. Priority levels are declared by default as
```
[:debug, :_, :default]
```
Where `:_` wildcards any other tag